common-lib/src/cmr/common/validations/json_schema.clj

```diff
@@ -7,102 +7,64 @@
    [cmr.common.log :as log :refer (warn info)]
    [clojure.string :as str])
   (:import
-   (com.github.fge.jsonschema.main JsonSchema
-                                   JsonSchemaFactory)
-   (com.github.fge.jackson JsonLoader)
-   (com.github.fge.jsonschema.core.report ListProcessingReport)
-   (com.fasterxml.jackson.core JsonParseException)))
+   (org.everit.json.schema Schema
```

Comment:
The original code wasn't following our new guide for namespaces: https://stuartsierra.com/2016/clojure-how-to-ns.html
```
(org.everit.json.schema Schema ValidationException)
```
is our preferred style now. There's a fewer other namespaces in this PR that should also be updated.